src/main/java/uk/gov/hmcts/reform/civil/utils/DocmosisTemplateDataUtils.java

```diff
@@ -55,19 +59,23 @@ public static String fetchRespondentName(CaseData caseData) {
     public static String fetchApplicantName(CaseData caseData) {
         StringBuilder applicantNameBuilder = new StringBuilder();
 
-        if (caseData.getApplicant2() != null) {
+        if (caseData.getApplicant1() != null && caseData.getApplicant2() != null) {
             applicantNameBuilder.append("1 ");
             applicantNameBuilder.append(caseData.getApplicant1().getPartyName());
             soleTraderCompany(caseData.getApplicant1(), applicantNameBuilder);
             applicantNameBuilder.append(" & 2 ");
             applicantNameBuilder.append(caseData.getApplicant2().getPartyName());
             soleTraderCompany(caseData.getApplicant2(), applicantNameBuilder);
-        } else {
+        } else if (caseData.getApplicant1() != null) {
             applicantNameBuilder.append(caseData.getApplicant1().getPartyName());
             soleTraderCompany(caseData.getApplicant1(), applicantNameBuilder);
             litigationFriend(caseData.getApplicant1LitigationFriend(), applicantNameBuilder);
+        } else {
+            String errorMsg = String.format("Applicant1 not found for claim number: "
+                                                + caseData.getCcdCaseReference());
+            log.error(errorMsg);
+            throw new IllegalArgumentException(errorMsg);
         }
-
```

Comment:
seems like you have removed this line.

Reply:
Which one? 
![image](https://user-images.githubusercontent.com/94067802/208708446-f00e81d5-3533-4b6b-ba7f-f837461bb5f8.png)